daemon/mgr/container_exec.go

```diff
@@ -77,7 +77,6 @@ func (mgr *ContainerManager) StartExec(ctx context.Context, execid string, attac
 			exitCode := 126
 			execConfig.ExitCode = int64(exitCode)
 		}
-		mgr.ExecProcesses.Put(execid, execConfig)
```

Comment:
For this part, I want to raise proposal that we should deep copy data from manager data store to avoid the data race. But it will introduce the complicated.